mmcv/image/io.py

```diff
@@ -91,7 +94,7 @@ def imread(img_or_path, flag='color', channel_order='bgr'):
                 cv2.cvtColor(img, cv2.COLOR_BGR2RGB, img)
             return img
     else:
-        raise TypeError('"img" must be a numpy array or a filename')
+        raise TypeError('"img" must be a numpy array, a str or a pathlib.Path object')
 
 
```

Comment:
The length of this line is also conflict with the requirement in flake8. You may change it to:
```python
raise TypeError('"img" must be a numpy array, a str or '
                'a pathlib.Path object')
```